app/environment.py

```diff
@@ -21,22 +22,27 @@ def display_env_state(env_state: EnvironmentState):
         st.table(env_state.nodes_table)
         col_net, col_links = st.columns([3, 2])
         with col_links:
-            st.table(env_state.traffic_table)
+            st.table(
+                env_state.traffic_table
+            )  # what the fuck is going on with the observation space changes and the LLM seeing these changes until the next step? Shits bugged...
```

Comment:
can you clarify this comment a bit

Reply:
Sometimes I break my code with aggressive comments so that I can remember where exactly I was when I left off... on this occasion I commented it out before I made the PR by mistake xD